packages/core/src/modules/oob/OutOfBandModule.ts

```diff
@@ -218,36 +218,63 @@ export class OutOfBandModule {
       )
     }
 
+    const outOfBandRecord = new OutOfBandRecord({
+      role: OutOfBandRole.Receiver,
+      state: OutOfBandState.PrepareResponse,
+      outOfBandMessage: outOfBandMessage,
+      autoAcceptConnection: autoAcceptConnection ?? this.agentConfig.autoAcceptConnections,
```

Comment:
For the connections record we don't store the agentConfig in the connection record. If we change the auto accept config parameter it should be reflected on all flows that don't have it expliclitly set. So I think to be consistent we should just use the autoAcceptConnection value (which may be undefined).

Reply:
I see. 👍